### DISCUSSION TITLE
Support for passive and offline scan

### DISCUSSION THREAD

**Discussion by dogasantos** [projectdiscovery/nuclei#882] [Ideas]
Hey guys!

me again with more ideas to share.
What you guys think about adding the power of nuclei templates into proxify? I think it would be fantastic!

it would be possible to add a second layer of vuln detection on dalfox scanning process for example. Add more customization to those tools and also have nuclei whole own set of passive templates.

also, we can take advantage of proxify dump files in order to apply scanning templates on it. That also will be powerful resource to catch details and passive vulns even without the need to send new requests to the target.

What you guys think?
Thanks

**Comment by ehsandeep** [comment]
@dogasantos nice ideas and a few of things already supported, for example, passive scan on HTTP response data - https://nuclei.projectdiscovery.io/nuclei/get-started/#passive-scan

Also moving this to discussion as in that way we can keep issues separate from discussion/ideas

**Reply by ehsandeep** [reply]
Going forward, we wanted to make use of this passive engine to integrate with **proxify**, to find passive issues as you go or browse through websites with proxify in between.

**Reply by dogasantos** [reply]
Fantastic!!

**Comment by BADERbad** [comment]
I am having difficulty running two tools, Nuclei, Proxify. What is the correct way to scan a site with these tools?

**Reply by MetzinAround** [reply]
Hi @BADERbad. 

Please open a discussion/issue specific to your difficulty in the correct repo or join us on discord (https://nux.gg/discord) to see if we can help you. Make sure to provide as many details as possible about your difficulty when asking for help.
